test/server/core/card/CardHighlighting.spec.ts

```diff
@@ -0,0 +1,67 @@
+describe('Card highlighting', function() {
```

Comment:
Let's rename this file to be like core/gameSteps/phases/ActionPhase.spec.ts, since it's testing some pieces of the action phase. "Card highlighting" is pretty generic since cards will be highlighted in the FE basically any time a selection prompt happens

Reply:
yessir!